package.json

```diff
@@ -39,7 +39,7 @@
     "eslint-webpack-plugin": "^2.5.4",
     "file-loader": "^6.2.0",
     "mini-css-extract-plugin": "^1.6.0",
-    "webpack": "^5.38.1",
+    "webpack": "^5.39.0",
```

Comment:
Is this version bump intentional, or just a side effect of some `npm` command run?

Reply:
I don't believe the version bump was intentional, but I do remember @farooqashar saying something about changing the `webpack` version to try and fix some bug..?

Reply:
Yes, there was an issue with the file `webpack-stats.json`not being generated after vising the browser and a workable solution was to downgrade the `webpack-bundle-tracker` package. I believe I did go back to the correct version for that package, but that automatically likely changed the webpack version as well. The version bump was not intentional, but was a side effect of downgrading and then upgrading the `webpack-bundle-tracker` package locally it seems like.